### DISCUSSION TITLE
[FEATURE] ARAC

### DISCUSSION THREAD

**Discussion by darkmoi31** [azerothcore/azerothcore-wotlk#3927] [Ideas]
Hello, I post this topic because I would like to know if someone has a SQL All Race All Class patch.
I have been looking for a few days for this.
thank you in advance

**Comment by BarbzYHOOL** [comment]
You could adapt those released long ago on ac-web

**Comment by darkmoi31** [comment]
Can you help me for adapt this please ... i have try but unsuccessfull...

**Comment by ghost** [comment]
Finally works ;)
All Class All Race as now available :D

![image](https://user-images.githubusercontent.com/58987080/73136074-ccbdc300-4049-11ea-9584-2432eccaee31.png)

![image](https://user-images.githubusercontent.com/58987080/73136075-cfb8b380-4049-11ea-9a6d-b6a29d947631.png)
[ARAC_AzerothCore.zip](https://github.com/azerothcore/forum/files/4113507/ARAC_AzerothCore.zip)

**Comment by Helias** [comment]
Wonderful, you could make a module for AC on this containing only SQL files, the DBC should be host in another repo/organization.

May we can make a gitlab repo and a mirror here for the catalogue (?), because we can't host DBC or client data here.

**Comment by Helias** [comment]
I think that we can close this thread because @iThorgrim-hub released a module for this!
it is available [here](https://github.com/azerothcore/mod-arac) and in the [AzerothCore catalogue](http://www.azerothcore.org/modules-catalogue/).
